docs/optics.md

```diff
@@ -8,7 +8,7 @@ circe provides support for optics by integrating with [Monocle](https://julien-t
 dependency on `circe-optics` to your build:
 
 ```scala
-libraryDependencies += "io.circe" %% "circe-optics" % circeVersion
+libraryDependencies += "io.circe" %% "circe-optics" % "@VERSION@"
```

Comment:
```suggestion
libraryDependencies += "io.circe" %% "circe-optics" % "0.15.0"
```

Reply:
used the version we use in build.sbt to add the dependency for docs. It inlines as 0.15.0:

<img width="822" alt="image" src="https://github.com/user-attachments/assets/0b3d3467-56e1-4e16-8b49-53620f79fc42">